docs/ocis/development/testing.md

```diff
@@ -476,14 +468,18 @@ As `antivirus` service is not enabled by default we need to enable the service w
 # run oCIS
 PROXY_ENABLE_BASIC_AUTH=true \
 ANTIVIRUS_SCANNER_TYPE="clamav" \
-ANTIVIRUS_CLAMAV_SOCKET="/var/run/clamav/clamd.ctl" \
+ANTIVIRUS_CLAMAV_SOCKET="tcp://host.docker.internal:3310" \
 POSTPROCESSING_STEPS="virusscan" \
 OCIS_ASYNC_UPLOADS=true \
 OCIS_ADD_RUN_SERVICES="antivirus"
 ocis/bin/ocis server
 ```
 {{< hint info >}}
 The value for `ANTIVIRUS_CLAMAV_SOCKET` is an example which needs adaption according your OS. If you are running `clamAv` with docker as per this documentation check the path that you mounted the volume i.e. `/your/local/filesystem/path/` to make sure the socket exists and give the full path to socket i.e. `/your/local/filesystem/path/clamd.sock` to `ANTIVIRUS_CLAMAV_SOCKET`.
```

Comment:
I didn't notice this earlier but I think this can be removed too
```suggestion
The value for `ANTIVIRUS_CLAMAV_SOCKET` is an example which needs adaption according your OS.
```